UPGRADE-5.0.md

```diff
@@ -11,3 +11,9 @@ Upgrading `bootstrap_form` to version 5 means you must upgrade your whole applic
 ## No `role="form"` Attribute
 
 As explained in #560, the `role="form"` attribute generated by `bootstrap_4` caused the W3C validator to output a warning. The `role="form"` attribute was deprecated in the 4.5.0 and is being remove completely in 5.0.0. This has no impact on `bootstrap_form` code itself, but may affect your application if it depended on a form having this attribute set. (Issue #569)
+
+## Different behavior for `errors_on` helper
+
+The `errors_on` helper now wraps the error message in a CSS class `invalid-feedback`, instead of `alert` and `alert-danger`, as before.
+
+This will display the error as any other [Bootstrap inline form error](https://getbootstrap.com/docs/5.0/forms/validation/#server-side), instead of displaying it as an [Bootstrap alert](https://getbootstrap.com/docs/5.0/components/alerts/).
```

Comment:
What do you think about adding a short note here to tell people what to do if they need the old behaviour? In other words, tell them about the option they can now provide to `errors_on` to change the class name(s). 

If you add a description to the README, you could even just redirect them from here to the relevant documentation in the README.

Reply:
This should indeed be documented as well.